register/forms.py

```diff
@@ -123,7 +123,7 @@ class Meta:
             "input_password",
             "confirm_password",
         )
-        exclude = ["password"]
+        exclude = ["password", "school"]

```

Comment:
If you're excluding this field, how are we getting the associated school information for admin staff during registration in the UI?

Reply:
I saw your "register/views.py" and there's a hard code school value (something you mentioned on slack as well), so you can add a TODO here for adding school as a form field (searchable) which provides the school.dbn to the admin_staff model for a later sprint.